src/inmanta/server/server.py

```diff
@@ -101,6 +107,32 @@ def start(self):
     @gen.coroutine
     def stop(self):
         yield super().stop()
+        yield self._close_resource_action_logger()
+
+    @gen.coroutine
+    def _create_resource_action_logger(self):
+        resource_action_log = os.path.join(opt.log_dir.get(), opt.server_resource_action_log.get())
+        self._file_handler = logging.handlers.WatchedFileHandler(filename=resource_action_log, mode='a+')
+        formatter = logging.Formatter(fmt="%(asctime)s %(levelname)-8s %(message)s")
+        self._file_handler.setFormatter(formatter)
+        self._file_handler.setLevel(logging.DEBUG)
+        self._resource_action_logger = logging.getLogger("RESOURCE_ACTION_LOG_FILE")
```

Comment:
looks a lot like a constant and should probably be a constant

the name of the logger is not that well chose, something like "resource_action_logger" for example would better describe it.